### PR TITLE
Add podspec for CocoaPods users

### PR DESCRIPTION
**Pull request by nwest**

### CODE REVIEW

File: OctoKit.swift.podspec [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  s.name             = "OctoKit.swift"
+  s.version          = "0.1.0"
+  s.summary          = "Swift version of the Octokit family"
+  s.description      = <<-DESC
+                        You are looking at the Swift port of the
+                        Octokit family. This is very unofficial and not maintained
+                        by Github.
+  DESC
+  s.homepage         = "https://github.com/nerdishbynature/octokit.swift"
+  s.license          = "MIT"
```
Comment (pietbrauer) [inline OctoKit.swift.podspec]: Can you link to to the file as well? I missed that in the last PR https://github.com/nerdishbynature/RequestKit/commit/6c1839ec97007b7380f1e45c86507e82ef4aa276

**Comment by nwest** [comment]
eek, looks like this is failing, related to https://github.com/Homebrew/homebrew/issues/45616

File: OctoKit.swift.podspec [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name             = "OctoKit.swift"
+  s.version          = "0.1.0"
```
Comment (pietbrauer) [inline OctoKit.swift.podspec]: Almost forgot: can you set the version to 0.4.0 (which is the current version)?

Comment (nwest) [inline OctoKit.swift.podspec]: Good catch!

**Comment by nwest** [comment]
Is there a way to retrigger codecov?

**Pull request merged**